FCCEUD-57: Review and align search instructions (#609)
redhataccess/pantheon: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-search-view-results.adoc

= Viewing search results

[role="_abstract"]
{ProductName} displays search results in two tables, *Modules* and *Assemblies*. which have the following columns:

* *Title*
* *Repository* : the repository in which content is located
* *Upload date* : the date on which the latest draft of the content was uploaded
* *Last Published date* : the date the content was last published

You can select the number of results you want to be displayed per page.

== Viewing content details

You can see additional details for each content item that is displayed in your search results.

.Procedure

> In the Search results table, click the link of a content item.

Reviewer: ```suggestion
In the Search results table, click an item.
```

+
{ProductName} displays the following information:

* *Module*: Complete directory path of the module's .adoc file.
* *Product*
* *Module type*
* *First published date*
* *Last published date*

== Previewing text

> You can see the text of the latest draft of a module or an assembly using the {ProductName} the search engine.

Reviewer 2: This statement does not sound very accurate to me. I would suggest something in the lines of 
"You can preview how the latest draft of a module or assembly will be rendered on the Customer Portal using {ProductName}"

Author: Good suggestion; thanks.

.Procedure

> . In the Search results table, click the link of a content item.

Reviewer: ```suggestion
. In the Search results table, click an item.
```
